packages/client/test/net/peer/rlpxpeer.spec.ts

```diff
@@ -29,6 +29,8 @@ describe('[RlpxPeer]', async () => {
       id: 'abcdef0123',
       host: '10.0.0.1',
       port: 1234,
+      address: 'address0',
```

Comment:
This is one of those weird artifacts of previously having had a libp2p transport option.  The RlpxPeer doesn't actually use the `address` string at all and I don't think it's a used element anywhere in our code.  I think we can safely delete this line since the `RlpxPeer` constructor creates this value and passes it to the `Peer` constructor.